docs/reference/elasticsearch/mapping-reference/semantic-text.md

```diff
@@ -9,863 +9,56 @@ applies_to:
-index that uses the `semantic_text` field.
-
-```console
-PUT my-index-000003
-{
-  "mappings": {
-    "properties": {
-      "inference_field": {
-        "type": "semantic_text",
-        "inference_id": "my-elser-endpoint-for-ingest",
-        "search_inference_id": "my-elser-endpoint-for-search"
-      }
-    }
-  }
-}
-```
-% TEST[skip:Requires inference endpoint]
-
-### Using ELSER on EIS
-
-```{applies_to}
-stack: preview 9.1, ga 9.2
-deployment:
-  self: unavailable
-serverless: ga
-```
-
-If you use the preconfigured `.elser-2-elastic` endpoint that utilizes the ELSER model as a service through the Elastic Inference Service ([ELSER on EIS](docs-content://explore-analyze/elastic-inference/eis.md#elser-on-eis)), you can
-set up `semantic_text` with the following API request:
-
-:::::::{tab-set}
-
-::::::{tab-item} Using ELSER on EIS on Serverless
-
-```{applies_to}
-serverless: ga
-```
+The following example creates an index mapping with a `semantic_text` field:
 
 ```console
-PUT my-index-000001
+PUT semantic-embeddings 
 {
-  "mappings": {
+  "mappings": { <1>
     "properties": {
-      "inference_field": {
-        "type": "semantic_text"
-      }
-    }
-  }
-}
-```
-
-::::::
-
-::::::{tab-item} Using ELSER on EIS in Cloud
-
-```{applies_to}
-stack: ga 9.2
-deployment:
-  self: unavailable
-```
-
-```console
-PUT my-index-000001
-{
-  "mappings": {
-    "properties": {
-      "inference_field": {
-        "type": "semantic_text",
-        "inference_id": ".elser-2-elastic"
+      "content": { 
+        "type": "semantic_text" 
       }
     }
   }
 }
 ```
 % TEST[skip:Requires inference endpoint]
 
-::::::
-
-:::::::
-
-## Parameters for `semantic_text` fields [semantic-text-params]
-
-`inference_id`
-:   (Optional, string) {{infer-cap}} endpoint that will be used to generate
-embeddings for the field. By default, `.elser-2-elasticsearch` is used.
-Use the [Create {{infer}} API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-inference-put)
-to create the endpoint. If `search_inference_id` is specified, the {{infer}}
-endpoint will only be used at index time.
-
-::::{applies-switch}
-
-:::{applies-item} { "stack": "ga 9.0" }
-This parameter cannot be updated.
-:::
-
-:::{applies-item} { "stack": "ga 9.3" }
-
-You can update this parameter by using
-the [Update mapping API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-mapping).
-You can update the inference endpoint if no values have been indexed or if the new endpoint is compatible with the current one.
-
-::::{warning}
-When updating an `inference_id` it is important to ensure the new {{infer}} endpoint produces embeddings compatible with those already indexed. This typically means using the same underlying model.
-::::
+1. In this example, there is no `inference_id` is specified, so `semantic_text` uses a [default inference endpoint](./semantic-text-how-tos.md#default-and-preconfigured-endpoints).
```

Comment:
```suggestion
1. In this example, the `semantic_text` field uses a [default inference endpoint](./semantic-text-how-tos.md#default-and-preconfigured-endpoints) because the `inference_id` parameter isn't specified.
```